From 0573ebae2185e98e0ef3047563f3e1e461ee7e8d Mon Sep 17 00:00:00 2001
From: Starwey <moonbite233@gmail.com>
Date: Thu, 24 Sep 2026 17:12:40 +0800
Subject: [PATCH] device/rockchip: AMP defconfig: enable RK_ROOTFS

Config.in.rootfs defaults RK_ROOTFS to 'n' when RK_AMP=y, so the rootfs was not built and update.img packing failed on a missing rootfs.img. Set RK_ROOTFS=y explicitly (as the Rockchip/Forlinx AMP defconfigs do).
---
 .chips/rk3576/LubanCat_rk3576_buildroot_amp_mcu_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.chips/rk3576/LubanCat_rk3576_buildroot_amp_mcu_defconfig b/.chips/rk3576/LubanCat_rk3576_buildroot_amp_mcu_defconfig
index 3196c31..7f8e883 100644
--- a/.chips/rk3576/LubanCat_rk3576_buildroot_amp_mcu_defconfig
+++ b/.chips/rk3576/LubanCat_rk3576_buildroot_amp_mcu_defconfig
@@ -1,3 +1,4 @@
+RK_ROOTFS=y
 RK_BUILDROOT_BASE_CFG="rk3576_lubancat"
 # RK_YOCTO is not set
 RK_ROOTFS_HOSTNAME_CUSTOM=y
-- 
2.55.0

